phonebook/PhoneDirectory.java

```diff
@@ -0,0 +1,47 @@
+package phonebook;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PhoneDirectory{
+    private  List<Record> phonebook;
+
+    public void add(String name, String phone) {
+        phonebook.add(new Record(name, phone));
+    }
+
+    public Record find(String name) {
+        Record record = null;
+
+        for (Record r : phonebook) {
+            if (r.getName().equals(name)) {
+                record = r;
+                break;
+            }
+        }
+        return record;
```

Comment:
Так будет короче.
Но null возвращать плохо.
```suggestion
        for (Record r : phonebook) {
            if (r.getName().equals(name)) {
                return r;
            }
        }
        return null;
```